git/week1/startupname.js

```diff
@@ -0,0 +1,8 @@
+let firstWords=["Easy","Awesome","Corporate","ToneRose","super","GrayMore","Brutaller","Voolzie","DoomDust","good"];
+let secondWords=["brillent","verygood","extraordinary","jems","hard","bad","worest","Lovelypot","Hipsterion","Prophicient"];
+let startupName ;
+const randomNumber = Math.floor(Math.random(startupName) * 10) + 0
```

Comment:
You're using one random index. This will work indeed, but what if two random indices were used, one for each array? How many more combinations of startup names will this allow you to generate?